### PR TITLE
openfst: enable more features

### PR DESCRIPTION
**Pull request by Mic92**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/openfst/default.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, python3, ncurses, lib, autoreconfHook }:
```
Comment (prusnak) [inline pkgs/development/libraries/openfst/default.nix:1]: Is `lib` really needed?
```suggestion
{ stdenv, fetchurl, python3, ncurses, autoreconfHook }:
```

Comment (Mic92) [reply pkgs/development/libraries/openfst/default.nix:1]: maybe it was before.

Comment (prusnak) [reply pkgs/development/libraries/openfst/default.nix:1]: It was you who added `lib`, I probably confused you with the suggestion.

Comment (Mic92) [reply pkgs/development/libraries/openfst/default.nix]: I meant, I maybe used it before in the expression and removed it later during refactoring. Drop it now.

**Review by prusnak — APPROVED** [review]

**Comment by Mic92** [comment]
I dropped python3 support for now. My patch was incomplete because it also requires a `toPythonPackage` call in `python-packages.nix` to be consumable. I don't need it right now, so I just skipped it.

**Comment by Mic92** [comment]
@GrahamcOfBorg build openfst

**Comment by Mic92** [comment]
cc @mweinelt this is what it takes to run rhasspy

**Comment by Mic92** [comment]
Builds times out on our macOS builder.

**Pull request merged**